packages/blade/src/components/DatePicker/__tests__/DatePicker.test.stories.tsx

```diff
@@ -189,7 +189,7 @@ DatePickerSingleSelectControlled.play = async () => {
   await expect(input).toHaveValue(dayjs().add(5, 'day').format('DD/MM/YYYY'));
   // select another date
   const dateToSelect = dayjs().add(2, 'day');
-  const date = getByRole('button', { name: dateToSelect.format('DD MMMM YYYY') });
+  const date = getByRole('button', { name: dateToSelect.format('D MMMM YYYY') });
```

Comment:
so if date is 1 may 2025, it will output 01 may 2025.... but our buttons will have 1 may 2025.. so these tests were breaking...